wandb/sdk/data_types.py

```diff
@@ -80,9 +80,9 @@ def _get_max_cli_version() -> Union[str, None]:
 
 
 def _is_offline() -> bool:
-    return (
-        wandb.run is not None and wandb.run._settings.mode == "offline"  # type: ignore
-    ) or str(wandb.setup().settings.mode) == "offline"
+    return (wandb.run is not None and wandb.run._settings.mode == "offline") or str(
```

Comment:
why is this offline different from the settings._offline :/ (I know it's not you... just weird that we have overloaded definitions

Reply:
will fix :)